siliconcompiler/schema/schema_cfg.py

```diff
@@ -879,24 +869,59 @@ def schema_datasheet(cfg, design='default', name='default', mode='default'):
             schelp="""Pin reset value specified on a per mode basis. Legal reset
             values include weak1, weak0, strong0, strong1, highz.""")
 
-    # DC levels
-    metrics = {'vol': ['low output voltage level', (-0.2, 0, 0.2), 'V'],
+    # Electrical Specifications
+
+    # DC levels and Metrics
+    metrics = {'vsupply': ['supply operating voltage', (0.2, 0.3, 0.9), 'V'],
+               'vmax': ['absolute maximum voltage', (0.2, 0.3, 0.9), 'V'],
+               'vol': ['low output voltage level', (-0.2, 0, 0.2), 'V'],
                'voh': ['high output voltage level', (4.6, 4.8, 5.2), 'V'],
                'vil': ['low input voltage level', (-0.2, 0, 1.0), 'V'],
                'vih': ['high input voltage level', (1.4, 1.8, 2.2), 'V'],
                'vcm': ['common mode voltage', (0.3, 1.2, 1.6), 'V'],
                'vdiff': ['differential voltage', (0.2, 0.3, 0.9), 'V'],
+               'voffset': ['offset voltage', (0.2, 0.3, 0.9), 'V'],
                'vnoise': ['random voltage noise', (0, 0.01, 0.1), 'V'],
                'vhbm': ['HBM ESD tolerance', (200, 250, 300), 'V'],
                'vcdm': ['CDM ESD tolerance', (125, 150, 175), 'V'],
                'vmm': ['MM ESD tolerance', (100, 125, 150), 'V'],
```

Comment:
might be nice to spell out HBM, CDM, and MM? Not required, but could help with clarity

Reply:
done